dao/src/main/java/cz.muni.fi.pa165.soccermanager/entity/Match.java

```diff
@@ -6,6 +6,47 @@
 @Entity
 public class Match {
 
+    public static class MatchBuilder {
+
+        private Date date;
+        private String stadium;
+        private final Team homeTeam;
+        private final Team awayTeam;
+        private boolean finished;
+        private int homeTeamGoals;
+        private int awayTeamGoals;
+
+        public MatchBuilder(Team homeTeam, Team awayTeam) {
+            this.homeTeam = homeTeam;
+            this.awayTeam = awayTeam;
+        }
+
+        public MatchBuilder date(Date date) {
+            this.date = date;
+            return this;
+        }
+
+        public MatchBuilder stadium(String stadium) {
+            this.stadium = stadium;
+            return this;
+        }
+
+        public MatchBuilder finished(boolean finished) {
+            this.finished = finished;
+            return this;
+        }
+
+        public MatchBuilder homeTeamGoals(int homeTeamGoals) {
+            this.homeTeamGoals = homeTeamGoals;
+            return this;
+        }
+
+        public MatchBuilder awayTeamGoals(int awayTeamGoals) {
+            this.awayTeamGoals = awayTeamGoals;
+            return this;
+        }
+    }
```

Comment:
You are missing build method here and constructor for Match class that takes builder as parameter so please add it. Sorry I also forget them in my code so if you used it as reference I've probably mistaken you